<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="A2MD20092" name="io.mosip.preregistration.tests.BatchJob" tests="2" failures="1" timestamp="03 Jul 2019 06:56:07 GMT" time="65.562" errors="0">
  <testcase name="preReg_BatchJob_batchJobForConsumedApplication" time="34.245" classname="io.mosip.preregistration.tests.BatchJob"/>
  <testcase name="preReg_BatchJob_batchJobForExpiredApplication" time="31.317" classname="io.mosip.preregistration.tests.BatchJob">
    <failure type="java.lang.AssertionError" message="Response Data Mismatch Failure  : expected is : Expired found is : Booked expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Response Data Mismatch Failure  : expected is : Expired found is : Booked expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at io.mosip.util.PreRegistrationLibrary.compareValues(PreRegistrationLibrary.java:997)
	at io.mosip.preregistration.tests.BatchJob.batchJobForExpiredApplication(BatchJob.java:88)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at io.mosip.testrunner.MosipTestRunner.startTestRunner(MosipTestRunner.java:42)
	at io.mosip.testrunner.MosipTestRunner.main(MosipTestRunner.java:30)
]]>
    </failure>
  </testcase> <!-- preReg_BatchJob_batchJobForExpiredApplication -->
</testsuite> <!-- io.mosip.preregistration.tests.BatchJob -->
